examples/text_labels_and_annotations/angles_on_bracket_arrows.py

```diff
@@ -0,0 +1,236 @@
+
+def get_point_of_rotated_vertical(origin, line_length, degrees):
+    """Return xy coordinates of the vertical line end rotated by degrees."""
+    rad = np.deg2rad(-degrees)
+    return [origin[0] + line_length * np.sin(rad),
+            origin[1] + line_length * np.cos(rad)]
+
+
+fig, ax = plt.subplots(figsize=(8, 7))
+ax.set(xlim=(0, 6), ylim=(-1, 4))
+ax.set_title('Angle annotations on bracket arrows')
+
+for i, stylename in enumerate(["]-[", "|-|"]):
+    for j, angle in enumerate([-40, 60]):
+        y = 2*i + j
+        arrow_centers = [(1, y), (5, y)]
+        vlines = [[c[0], y + 0.5] for c in arrow_centers]
+        widths = "widthA=1.5,widthB=1.5"
+        arrowstyle = f"{stylename},{widths},angleA={angle},angleB={-angle}"
+        patch = FancyArrowPatch(arrow_centers[0], arrow_centers[1],
+                                arrowstyle=arrowstyle, mutation_scale=25)
+        ax.add_patch(patch)
+        ax.text(3, y + 0.05, arrowstyle.replace(f"{widths},", ""),
+                verticalalignment="bottom", horizontalalignment="center")
+        ax.vlines([i[0] for i in vlines], [y, y], [i[1] for i in vlines],
+                  linestyles="--", color="C0")
+        # Get the coordinates for the drawn patches at A and B
+        patch_top_coords = [
+            get_point_of_rotated_vertical(arrow_centers[0], 0.5, angle),
+            get_point_of_rotated_vertical(arrow_centers[1], 0.5, -angle)
+        ]
+        # Create points for annotating A and B with AngleAnnotation
+        # Points include the top of the vline and patch_top_coords
+        pointsA = [(1, y + 0.5), patch_top_coords[0]]
+        pointsB = [patch_top_coords[1], (5, y + 0.5)]
+        # Define the directions for the arrows for AngleAnnotation
+        arrow_angles = [0.5, -0.5]
+        # Reverse the points and arrow_angles when the angle is negative
+        if angle < 0:
+            pointsA.reverse()
+            pointsB.reverse()
+            arrow_angles.reverse()
+        # Add AngleAnnotation and arrows to show angle directions
+        data = zip(arrow_centers, [pointsA, pointsB], vlines, arrow_angles,
+                   patch_top_coords)
+        for center, points, vline, arrow_angle, patch_top in data:
+            am = AngleAnnotation(center, points[0], points[1], ax=ax,
+                                 size=0.25, unit="axes min", text=str(-angle))
+            arrowstyle = "Simple, tail_width=0.5, head_width=4, head_length=8"
+            kw = dict(arrowstyle=arrowstyle, color="C0")
+            arrow = FancyArrowPatch(vline, patch_top,
+                                    connectionstyle=f"arc3,rad={arrow_angle}",
+                                    **kw)
+            ax.add_patch(arrow)
+
+
+#############################################################################
+#
+# .. admonition:: References
+#
+#    The use of the following functions, methods, classes and modules is shown
+#    in this example:
+#
+#    - `matplotlib.patches.ArrowStyle`
```

Comment:
I'm not sure why the mini gallery was not created from this? As per the [docs](https://matplotlib.org/devdocs/devel/documenting_mpl.html#references-for-sphinx-gallery) is this all that is required @story645 or it there some other reference required in the docstring?